dojos.js

```diff
@@ -2137,6 +2156,8 @@ module.exports = function (options) {
         client.end();
         _.each(results.rows, function (dojo) {
           dojo.user_invites = purgeInviteEmails(dojo.user_invites);
+          delete dojo.eventbriteToken;
+          delete dojo.eventbriteWhId;
```

Comment:
Maybe centralise and call to it instead of writing the same `delete`s every time?